src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1782,18 +1825,18 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 					break;
 			}
 
-			stickyTabsWidth = this.group.stickyCount * stickyTabWidth;
+			stickyTabsWidth = this.tabsModel.stickyCount * stickyTabWidth;
 		}
 
 		// Figure out if active tab is positioned static which has an
 		// impact on whether to reveal the tab or not later
-		let activeTabPositionStatic = this.accessor.partOptions.pinnedTabSizing !== 'normal' && this.group.isSticky(activeIndex);
+		let activeTabPositionStatic = this.accessor.partOptions.pinnedTabSizing !== 'normal' && activeTabIndex && this.tabsModel.isSticky(activeTabIndex);
 
```

Comment:
Careful, when you do `if (number)` it returns false for `0`, so you need to do `typeof activeTabIndex === 'number'`